query/language.peg

```diff
@@ -140,12 +140,10 @@ predicateClause <- "where" __ predicate_1
 # ...
 
 predicate_1 <-
-  predicate_2 OP_AND predicate_1 { p.addAndPredicate() } /
-  predicate_2 /
+  predicate_2 (OP_AND predicate_2 { p.addAndPredicate() })*
 
 predicate_2 <-
-  predicate_3 OP_OR predicate_2 { p.addOrPredicate() } /
-  predicate_3
+  predicate_3 (OP_OR predicate_3 { p.addAndPredicate() })*
```

Comment:
the `.addAndPredicate()` should probably be `.addOrPredicate()` to match `OP_OR`.